.infrastructure/55_cognito.tf

```
@@ -0,0 +1,117 @@
+resource "aws_cognito_user_pool" "devportal" {
+  name                = "devportalpool"
+  deletion_protection = "ACTIVE"
+
+  user_pool_add_ons {
+    advanced_security_mode = "OFF"
+  }
+
+  username_attributes      = ["email"]
+  auto_verified_attributes = ["email"]
+
+  account_recovery_setting {
+    recovery_mechanism {
+      name     = "verified_email"
+      priority = 1
+    }
+  }
+
+  admin_create_user_config {
+    # allow user to create via signup page
+    allow_admin_create_user_only = false
+  }
+
+  password_policy {
+    minimum_length = 8
+  }
+
+  email_configuration {
+    email_sending_account = "COGNITO_DEFAULT"
+  }
+
```

Add a comment with some information about attributes

```suggestion
  # Custom attribute cannot be required.
  # Terraform cannot update or delete an attribute.
  # Terraform can add new attribute as update in-place.
```